### ISSUE TITLE
Slug is always "untitled" after editing body first

### ISSUE THREAD

**Issue by kevinansfield** [TryGhost/Ghost#8482]
### Issue Summary

In LTS if the body of a new post is edited before a title is set we don't start auto-saving to avoid creating a slug until a save is forced or the title is edited. This means it's unlikely the slug needs to be set manually but there is a higher chance of losing content because there's no auto-save.

In the alpha we trigger a save as soon as the first character is entered in the post content resulting in a saved slug of `untitled` which can then only be updated manually.

### Steps to Reproduce

1. Click "New story" link
2. Click in the post body area
3. Start typing
4. Note that the title becomes `(Untitled)` and the slug becomes `untitled` (or `untitled-1/2/3` and so on)
5. Go back to the title and edit it
6. Note that the slug will stay as `untitled`

We should avoid saving `untitled` slugs where possible, I see two initial solutions here:

1. We return to the LTS behaviour (https://github.com/TryGhost/Ghost-Admin/issues/722)
2. The client detects when a slug matches the `untitled(-1/2/3)` pattern and triggers the normal slug generation process as it would when editing the title before the body

**Comment by kevinansfield** [comment]
Option 1 listed above was implemented in https://github.com/TryGhost/Ghost-Admin/pull/722, option 2  could still be implemented to improve the untitled slug situation even after a post has been saved as `untitled-x`

**Comment by kevinansfield** [comment]
Closing in favour of #8736
